apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -2005,6 +2009,25 @@ get_status_sut(IntAsString) ->
     Parameters = case IntAsString of true -> "?int-as-string"; false -> "" end,
     http_request(Host, get, "status" ++ Parameters, []).
 
+%% /recent-gas-prices
+
+get_recent_gas_prices(_Config) ->
+    Host = external_address(),
+    {ok, 200, [
+        #{ <<"blocks">> := 1, <<"min_gas_price">> := MinGasPrice1 },
+        #{ <<"blocks">> := 5, <<"min_gas_price">> := MinGasPrice5 },
+        #{ <<"blocks">> := 20, <<"min_gas_price">> := MinGasPrice20 },
+        #{ <<"blocks">> := 120, <<"min_gas_price">> := MinGasPrice120 },
+        #{ <<"blocks">> := 480, <<"min_gas_price">> := MinGasPrice480 }
+    ]} = http_request(Host, get, "recent-gas-prices", []),
+    MinGasPrice = 1000000000,
```

Comment:
Not very convincing test :)

Reply:
Hmm, no... Didn't look at it, just updated it to pass 😅